components/admin_console/brand_image_setting/brand_image_setting.tsx

```diff
@@ -78,6 +78,9 @@ export default class BrandImageSetting extends React.PureComponent<Props, State>
             } else {
                 this.setState({brandImageExists: false});
             }
+        }).catch(() => {
+            console.log('unable to retrieve brand image'); //eslint-disable-line no-console
```

Comment:
Can we make the following small modification?
```suggestion
        }).catch((err) => {
            console.error(`unable to retrieve brand image: ${err}`); //eslint-disable-line no-console
```

Reply:
Very good suggestion, thank you @hahmadia.  Without that, this change could have inadvertently suppressed fetch error messages on the console, I think; that wasn't my intent!